api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -53,23 +53,26 @@
 
     private Extension ext5;
 
-
     @Before
     public void setUp() throws Exception {
 
+    	// clear up from previous tests
```

Comment:
The reason DirtiesContext was added was to remove the apps and extension added by the test after it's done, so Darius is right you need to call these in a method annotated with @After, and i guess with that you won't need to change the tests